CMakeLists.txt

```diff
@@ -40,6 +42,12 @@ endif()
 add_subdirectory(src)
 add_subdirectory(run)
 
+# todo: probe for emcc
```

Comment:
Please remove this. I'll file an issue for it instead, so it doesn't get forgotten.